src/components/Checkout/DonationItem.vue

```diff
@@ -353,8 +353,9 @@ export default {
 						'basket',
 						'Update Donation',
 						'Update Success',
-						// pass donation amount as whole number
-						numeral(this.amount).value() * 100,
+						// pass raw donation amount to property field
+						numeral(this.amount).format('0,0.00'),
+						// pass donation amount as whole number in value field
```

Comment:
Should the value also be the raw donation?

Reply:
No, we pass an augmented value to ensure the google events can recieve it. Google only supports a whole number, not floats.

I've reverted this anyhow b/c it's converted so that when we see 1 it's 1 cent... which is correct, just confusing.